README.md

```diff
@@ -26,11 +26,48 @@ Todo:
 
 **Elaborate description of proposed devroom (including possible topics)**:
 
-Talks regarding the Nim programming language, similar to Nim Conferences in 2020 and 2021.
+Nim is a versatile language that can both be compiled to low-level languages
+like C/C++/ObjC and to the web-oriented language JavaScript. This, along with
+it's rich, static type system, tuneable and optional GC, and its rich
```

Comment:
*tunable* is the more widespread variant spelling. Perhaps, "optional" should be put first.